spec/pods.md

```diff
@@ -186,3 +187,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
 * **ports** (list of objects, optional) list of ports that SHOULD be exposed on the host.
     * **name** (string, required, restricted to the [AC Name](#ac-name-type) formatting) name of the port to be exposed on the host. This field is a key referencing by name ports specified in the Image Manifest(s) of the app(s) within this Pod Manifest; consequently, port names MUST be unique among apps within a pod.
     * **hostPort** (integer, required) port number on the host that will be mapped to the application port.
+    * **hostIP** (IP string, optional) IP address to forward from. If ommitted, all IPs will be used
+    * **podPort** (Object, optional) A Port object as specified in an Image Manifest. If supplied, this overrides any matching ports specified by the images.
```

Comment:
(object, optional) 
port object of the same structure as in an image manifest [see the "ports" field in the [Image Manifest Schema](aci.md#image-manifest-schema))